projects/plugins/jetpack/extensions/blocks/blogroll/blogroll-item/editor.scss

```diff
@@ -1,41 +1,63 @@
-.wp-block-jetpack-blogroll-item {
-	display: flex;
-	padding: 20px 8px;
-	gap: 20px;
+.wp-block-jetpack-blogroll {
 
-	font-style: normal;
-	font-weight: 400;
-
-		figure {
-			margin: 0;
-			height: 48px;
-			width: 48px;
+	h3 + div .jetpack-blogroll-item-divider {
+		display: none;
+	}
 
-			&.empty-site-icon {
-				background: url('../placeholder-site-icon.svg');
-				background-size: cover;
+	.jetpack-blogroll-item-divider {
+		margin-block-start: 0;
+		margin-block-end: 0;
+		border-top: 1px solid var(--wp--preset--color--foreground);
```

Comment:
There is no guarantee that we will have this color in the theme.
We could add the class `wp-block-separator` to the hr to ensure that the right color will be applied. Or we can render a separator block to which we apply the rules.

And we can keep: 			
	```
	border-width: 1px 0 0 0;
	border-style: solid;
	opacity: 0.2;
```